app/src/main/java/com/genymobile/pr/ui/PullRequestDetailsDialogFragment.java

```diff
@@ -0,0 +1,93 @@
+package com.genymobile.pr.ui;
+
+import com.genymobile.pr.R;
+import com.genymobile.pr.bus.BusProvider;
+import com.genymobile.pr.bus.IssueRetrievedEvent;
+import com.genymobile.pr.model.Issue;
+import com.genymobile.pr.model.Label;
+import com.genymobile.pr.net.GitHubProvider;
+import com.genymobile.pr.net.IssueCallback;
+
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v4.app.DialogFragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+import android.widget.LinearLayout.LayoutParams;
+import android.widget.TextView;
+import com.squareup.otto.Subscribe;
+
+// TODO add a loader when retrieving data.
+// TODO handle retrieving data errors
+public class PullRequestDetailsDialogFragment extends DialogFragment {
+    private static final String NUMBER = "number";
+    private static final String REPO = "repo";
+    private static final int DEFAULT_SPACING = 10;
+    int number;
+    String repo;
+
+    static PullRequestDetailsDialogFragment newInstance(int pullRequestNumber, String repo) {
```

Comment:
`public` not because it's really better but it's done in other `newInstance()` methods I think